src/components/TimeSeriesPanel/TimeSeriesChartHeader.tsx

```diff
@@ -51,6 +53,8 @@ import { makeStyles } from "@/util/styles";
 import { TimeSeriesChartType } from "@/states/controlState";
 import TimeSeriesAddButton from "./TimeSeriesAddButton";
 import ValueRangeEditor from "./ValueRangeEditor";
+import { exportElement, ExportOptions } from "@/util/export";
+import { postMessage } from "@/actions/messageLogActions";
```

Comment:
No imports from actions here please!

Action imports are only done in `src/connected`. Here we are in `src/components` which is action-free to stay reusable for later projects. Use property `postMessage` instead.